packages/devtools-reps/src/object-inspector/reducer.js

```diff
@@ -49,6 +50,21 @@ function reducer(
     return cloneState();
   }
 
+  if (type === "GETTER_INVOKED") {
+    return cloneState({
+      actors: data.actor
+        ? new Set(state.actors || []).add(data.result.from)
+        : state.actors,
+      evaluations: new Map(state.evaluations).set(data.node.path, {
+        timestamp: Date.now(),
```

Comment:
could we use the actor ID as the key or if we know there is just one evaluation just `-evaluation`